cuqi/distribution/_truncated_normal.py

```diff
@@ -0,0 +1,48 @@
+import numpy as np
+from scipy.special import erf
+from cuqi.distribution import Distribution
+
+class TruncatedNormal(Distribution):
+    """
+    Truncated Normal probability distribution. Generates instance of cuqi.distribution.TruncatedNormal.
+    
+    The variables of this distribution are iid.
+
+    
+    Parameters
```

Comment:
I suggest adding types to the parameters, e.g. 
```
    mean: scalar or 1d-array
        Mean of the distribution
```

Reply:
Done as in uniform:
```python
    mean : float or array_like of floats
        mean of distribution
    std : float or array_like of floats
        standard deviation
    low : float or array_like of floats
        lower bound of the distribution
    high : float or array_like of floats
        upper bound of the distribution
```

Reply:
👍